docs/interface.md

```diff
@@ -27,6 +27,14 @@ The following statuses are valid:
 - `disapprove_with_comment`: To be used when a solution can be disapproved as suboptimal and a comment is provided.
 - `refer_to_mentor`: This is the default situation and should be used when there is any uncertainty.
 
+## Debugging
+
+The contents of stdout and stderr from each run will be persisted into files that can be viewed later.
+
+You maybe write an `analysis.out` file that contains and debugging information you want to later view.
```

Comment:
For grammar: `maybe` -> `may`?

Reply:
or "You might write" if it is optional as indicated, rather than permission based.

Reply:
also remove the `and` from that sentence...
And switch the last words around `you want to (re)view later`.

At least that's what my non-native speaker feeling tells me ;)